scala3.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: scala3
+  version: 3.2.2
+  epoch: 0  
+  description: The scala compiler
+  copyright:
+      license: Apache-2.0
+  dependencies:
+    runtime:
+      - openjdk-17-jre
```

Comment:
Should this be something we enforce, or something the user who's installing scala also needs to be responsible for?

Like, if someone wants to run scala3 with openjdk-11-jre (or openjdk-19 or something), is that a valid configuration we should allow?